inputParser/flagParser/flag-parser.go

```diff
@@ -14,6 +14,7 @@ func Parse(ctx *model.CheckContext) *model.CheckContext {
 	password := flag.String("P", "", "Password of the service")
 	databaseName := flag.String("n", "", "Name of the database")
 	sslMode := flag.String("ssl", "disable", "Enable or Disable ssl mode (for some database or services)")
+	clusterMode := flag.String("cluster", "disable", "Enable or Disable cluster mode (for some database or services)")
```

Comment:
Cluster is something that is mostly a boolean thing, true or false.
But let's say we want to have sentinel in place, how would you propose the flag name without having a new one defined for sentinel?
can we somehow have cluster and sentinel in a single flag? maybe for Redis, we can have `operation-mode` flag and then we can have `single, sentinel, cluster` 
I mean like this:
`-operation-mode=cluster`

then this would be more general I would say and the flag can also be reused later on.